Classes/Domain/AbstractEventSourcedAggregateRoot.php

```diff
@@ -42,8 +42,8 @@
     public static function reconstituteFromEventStream(string $identifier, EventStream $stream)
     {
         $instance = new static($identifier);
-        foreach ($stream as $eventWithMetadata) {
-            $instance->apply($eventWithMetadata->getEvent());
+        foreach ($stream as $eventTransport) {
+            $instance->apply($eventTransport->getEvent());
```

Comment:
I'm still not happy with the name "eventTransport" because it doesn't communicate which transport this is about (from store to publisher).
Explanation: This is now a wrapper for the raw event (including version & metadata) from the EventStore + the converted event (see below)